docs/testing.rst

```diff
@@ -821,3 +881,4 @@ Utilities
 
 .. _pytest: http://pytest.org/latest/
 .. _pytest-aiohttp: https://pypi.python.org/pypi/pytest-aiohttp
+.. _asynctest: https://pypi.org/project/asynctest/
```

Comment:
Maybe link the docs instead? https://asynctest.readthedocs.io/en/latest/

Reply:
Better yet — link it via intersphinx @ https://github.com/aio-libs/aiohttp/blob/47996e5/docs/conf.py#L59-L73

Reply:
I did not know how to proceed, I saw that pytest had a website and with pytest-aiohttp pipit package was linked. Thanks to you now I know more about sphinx, it is a great idea to link to their documentation.

Reply:
How does it work?

Reply:
Basically, you need to link their RTD site in `docs/conf.py` and then reference things from their docs where they are mentioned.

It looks like there's an `index` document exposed:
<details>
<summary>
    <code>$ <kbd>python -m sphinx.ext.intersphinx https://asynctest.readthedocs.io/en/latest/objects.inv</kbd></code>
</summary>

```console
py:attribute
	asynctest.CoroutineMock.await_args       asynctest.mock.html#asynctest.CoroutineMock.await_args
	asynctest.CoroutineMock.await_args_list  asynctest.mock.html#asynctest.CoroutineMock.await_args_list
	asynctest.CoroutineMock.await_count      asynctest.mock.html#asynctest.CoroutineMock.await_count
	asynctest.CoroutineMock.awaited          asynctest.mock.html#asynctest.CoroutineMock.awaited
	asynctest.FileDescriptor.next_fd         asynctest.selector.html#asynctest.FileDescriptor.next_fd
	asynctest.NonCallableMock.is_coroutine   asynctest.mock.html#asynctest.NonCallableMock.is_coroutine
	asynctest.TestCase.forbid_get_event_loop asynctest.case.html#asynctest.TestCase.forbid_get_event_loop
	asynctest.TestCase.loop                  asynctest.case.html#asynctest.TestCase.loop
	asynctest.TestCase.use_default_loop      asynctest.case.html#asynctest.TestCase.use_default_loop
py:class
	asynctest.ClockedTestCase                asynctest.case.html#asynctest.ClockedTestCase
	asynctest.CoroutineMock                  asynctest.mock.html#asynctest.CoroutineMock
	asynctest.FileDescriptor                 asynctest.selector.html#asynctest.FileDescriptor
	asynctest.FileMock                       asynctest.selector.html#asynctest.FileMock
	asynctest.FunctionTestCase               asynctest.case.html#asynctest.FunctionTestCase
	asynctest.MagicMock                      asynctest.mock.html#asynctest.MagicMock
	asynctest.Mock                           asynctest.mock.html#asynctest.Mock
	asynctest.NonCallableMock                asynctest.mock.html#asynctest.NonCallableMock
	asynctest.SSLSocketMock                  asynctest.selector.html#asynctest.SSLSocketMock
	asynctest.SocketMock                     asynctest.selector.html#asynctest.SocketMock
	asynctest.TestCase                       asynctest.case.html#asynctest.TestCase
	asynctest.TestSelector                   asynctest.selector.html#asynctest.TestSelector
py:data
	asynctest.GLOBAL                         asynctest.mock.html#asynctest.GLOBAL
	asynctest.LIMITED                        asynctest.mock.html#asynctest.LIMITED
py:function
	asynctest.create_autospec                asynctest.mock.html#asynctest.create_autospec
	asynctest.fail_on                        asynctest.case.html#asynctest.fail_on
	asynctest.fd                             asynctest.selector.html#asynctest.fd
	asynctest.helpers.exhaust_callbacks      asynctest.helpers.html#asynctest.helpers.exhaust_callbacks
	asynctest.ignore_loop                    asynctest.case.html#asynctest.ignore_loop
	asynctest.isfilemock                     asynctest.selector.html#asynctest.isfilemock
	asynctest.lenient                        asynctest.case.html#asynctest.lenient
	asynctest.mock_open                      asynctest.mock.html#asynctest.mock_open
	asynctest.patch                          asynctest.mock.html#asynctest.patch
	asynctest.patch.dict                     asynctest.mock.html#asynctest.patch.dict
	asynctest.patch.multiple                 asynctest.mock.html#asynctest.patch.multiple
	asynctest.patch.object                   asynctest.mock.html#asynctest.patch.object
	asynctest.return_once                    asynctest.mock.html#asynctest.return_once
	asynctest.set_read_ready                 asynctest.selector.html#asynctest.set_read_ready
	asynctest.set_write_ready                asynctest.selector.html#asynctest.set_write_ready
	asynctest.strict                         asynctest.case.html#asynctest.strict
py:method
	asynctest.ClockedTestCase.advance        asynctest.case.html#asynctest.ClockedTestCase.advance
	asynctest.CoroutineMock.assert_any_await asynctest.mock.html#asynctest.CoroutineMock.assert_any_await
	asynctest.CoroutineMock.assert_awaited   asynctest.mock.html#asynctest.CoroutineMock.assert_awaited
	asynctest.CoroutineMock.assert_awaited_once asynctest.mock.html#asynctest.CoroutineMock.assert_awaited_once
	asynctest.CoroutineMock.assert_awaited_once_with asynctest.mock.html#asynctest.CoroutineMock.assert_awaited_once_with
	asynctest.CoroutineMock.assert_awaited_with asynctest.mock.html#asynctest.CoroutineMock.assert_awaited_with
	asynctest.CoroutineMock.assert_has_awaits asynctest.mock.html#asynctest.CoroutineMock.assert_has_awaits
	asynctest.CoroutineMock.assert_not_awaited asynctest.mock.html#asynctest.CoroutineMock.assert_not_awaited
	asynctest.CoroutineMock.reset_mock       asynctest.mock.html#asynctest.CoroutineMock.reset_mock
	asynctest.FileMock.fileno                asynctest.selector.html#asynctest.FileMock.fileno
	asynctest.TestCase.addCleanup            asynctest.case.html#asynctest.TestCase.addCleanup
	asynctest.TestCase.assertAsyncRaises     asynctest.case.html#asynctest.TestCase.assertAsyncRaises
	asynctest.TestCase.assertAsyncRaisesRegex asynctest.case.html#asynctest.TestCase.assertAsyncRaisesRegex
	asynctest.TestCase.assertAsyncWarns      asynctest.case.html#asynctest.TestCase.assertAsyncWarns
	asynctest.TestCase.assertAsyncWarnsRegex asynctest.case.html#asynctest.TestCase.assertAsyncWarnsRegex
	asynctest.TestCase.doCleanups            asynctest.case.html#asynctest.TestCase.doCleanups
	asynctest.TestCase.setUp                 asynctest.case.html#asynctest.TestCase.setUp
	asynctest.TestCase.tearDown              asynctest.case.html#asynctest.TestCase.tearDown
	asynctest.TestSelector.close             asynctest.selector.html#asynctest.TestSelector.close
	asynctest.TestSelector.modify            asynctest.selector.html#asynctest.TestSelector.modify
	asynctest.TestSelector.register          asynctest.selector.html#asynctest.TestSelector.register
	asynctest.TestSelector.select            asynctest.selector.html#asynctest.TestSelector.select
	asynctest.TestSelector.unregister        asynctest.selector.html#asynctest.TestSelector.unregister
py:module
	asynctest                                index.html#module-asynctest
	asynctest.case                           asynctest.case.html#module-asynctest.case
	asynctest.helpers                        asynctest.helpers.html#module-asynctest.helpers
	asynctest.mock                           asynctest.mock.html#module-asynctest.mock
	asynctest.selector                       asynctest.selector.html#module-asynctest.selector
std:doc
	asynctest.case                           Module case                             : asynctest.case.html
	asynctest.helpers                        Module helpers                          : asynctest.helpers.html
	asynctest.mock                           Module mock                             : asynctest.mock.html
	asynctest.selector                       Module selector                         : asynctest.selector.html
	examples/index                           List of code examples                   : examples/index.html
	examples/tutorial/clock.py               tutorial/clock.py                       : examples/tutorial/clock.py.html
	examples/tutorial/mocking.py             tutorial/mocking.py                     : examples/tutorial/mocking.py.html
	examples/tutorial/mocking_io.py          tutorial/mocking_io.py                  : examples/tutorial/mocking_io.py.html
	examples/tutorial/patching.py            tutorial/patching.py                    : examples/tutorial/patching.py.html
	examples/tutorial/test_cases.py          tutorial/test_cases.py                  : examples/tutorial/test_cases.py.html
	index                                    asynctest documentation                 : index.html
	tutorial.advanced                        Advanced Features                       : tutorial.advanced.html
	tutorial.introduction                    Introduction                            : tutorial.introduction.html
	tutorial.mocking                         Mocking                                 : tutorial.mocking.html
	tutorial.test_cases                      Test cases                              : tutorial.test_cases.html
std:label
	genindex                                 Index                                   : genindex.html
	modindex                                 Module Index                            : py-modindex.html
	search                                   Search Page                             : search.html
```
</details>
_(http://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#showing-all-links-of-an-intersphinx-mapping-file)_

So in docs it'd be smth like `asynctest <asynctest:index>` if you call this docs set "asynctest".

Reply:
@WisdomPill will you implement this?

Reply:
Yes, I just prioritized the rest because I still have to read the docs on how to do it

Reply:
Made it work, it is pretty slick now, have a look